packages/neo-one-typescript-concatenator/src/Concatenator.ts

```diff
@@ -370,14 +416,20 @@ export class Concatenator {
       const existingNamedImports = tsUtils.importDeclaration.getNamedImports(existingImport);
       const existingNames = new Set(existingNamedImports.map((namedImport) => namedImport.name.text));
       const filteredImports = namedImports.filter((namedImport) => !existingNames.has(namedImport.name.text));
+      const concattedImports = existingNamedImports.concat(filteredImports);
 
       this.sourceFileToImports.set(
         file,
         tsUtils.setOriginalRecursive(
           ts.createImportDeclaration(
             undefined,
             undefined,
-            ts.createImportClause(undefined, ts.createNamedImports(existingNamedImports.concat(filteredImports))),
+            ts.createImportClause(
+              defaultImport === undefined ? undefined : this.getIdentifierForIdentifier(defaultImport),
```

Comment:
This should use the existing one. I.e. `tsUtils.importDeclaration.getDefaultImport(existingImport);`, if it exists, otherwise the new one.